problems/enumerated/problem_10107/english_text.py

```diff
@@ -0,0 +1,8 @@
+ani_string = "Ani"
+mayrik_string  = "Mammy"
+one_year_after_string = "$1$ year later"
+now_string = "Այժմ"
```

Comment:
`Now` or `Currently, `Present`, depending on context?

Reply:
It's just I forgot to translate the word, I think `now` is perfect

Reply:
In that case, can you change this?